Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -670,6 +670,9 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
     virtual bool isGraphicsLayerTextureMapper() const { return false; }
     virtual bool isCoordinatedGraphicsLayer() const { return false; }
 
+    bool paintUsingCompositeCopy() const { return m_paintUsingCompositeCopy; }
```

Comment:
`canPaintUsingCompositeCopy()` or `shouldPaintUsingCompositeCopy()` (this function isn't doing a paint).